mix.exs

```diff
@@ -2,15 +2,23 @@ defmodule Changelog.Mixfile do
   use Mix.Project
 
   def project do
-    [app: :changelog,
-     version: "0.0.1",
-     elixir: "~> 1.3",
-     elixirc_paths: elixirc_paths(Mix.env),
-     compilers: [:phoenix] ++ Mix.compilers,
-     build_embedded: Mix.env == :prod,
-     start_permanent: Mix.env == :prod,
-     aliases: aliases,
-     deps: deps]
+    [
+      app: :changelog,
+      version: "0.0.1",
+      elixir: "~> 1.3",
+      elixirc_paths: elixirc_paths(Mix.env),
+      compilers: [:phoenix] ++ Mix.compilers,
+      build_embedded: Mix.env == :prod,
+      start_permanent: Mix.env == :prod,
+      aliases: aliases,
+      deps: deps,
+      preferred_cli_env: [
```

Comment:
I prefer single space because that's how the data structure works. It is also consistent with Elixir's own inspect :)

In any case, you probably shouldn't mix code changes with style changes. Better to dance the tango first and then start a separate discussion about style.